database/migrations/2024_05_10_110438_add_group_id_to_role.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('roles', function (Blueprint $table) {
+            $table->foreignId('group_id')->nullable();
```

Comment:
Ik weet niet precies hoe het komt, alleen er is geen correcte link in de db tussen roles en group(id)